### ISSUE TITLE
delay before ENTERED is triggered

### ISSUE THREAD

**Issue by powysm** [deckameron/Ti.Android.Geofence#9]
I am seeing a delay of up to 2-3 minutes on entering a fence, before the entered event is triggered. 
I have added in my own Ti.Geolocation event listener and can see the location changing as i move about (into a new fence), but the entered fences aren't triggered until much later. 

Is this related to battery consumption considerations or as intended for some other reason?
Or potentially its just me seeing this and its device/connection related, although the geolocation events would indicate not.

Thanks (again)
Mathew

**Comment by deckameron** [comment]
Hi @powysm , are you using Android 8+?
It is important to know that on Android 8 and 9 the geofencing has a latency of 2-3 minutes, due to some new limitations.

**Comment by powysm** [comment]
Yes 8.0.0. So this is purely OS related, therefore other similar Ti modules
would have the same problem?
Thanks for letting me know.

**Comment by deckameron** [comment]
@powysm I believe so! 
I am still trying to understand a few nuances of this limitations. I believe there is a way to make work faster that I need to find out.
